R/plumber.R

```diff
@@ -299,7 +307,15 @@ Plumber <- R6Class(
         path <- paste0(path, "/")
       }
 
-      private$mnts[[path]] <- router
+      # Remove prior mount if it exists
+      self$unmount(path)
```

Comment:
I don't think you want to do this unmount...? With https://github.com/rstudio/plumber/pull/883, you'll be able to mount two subrouters to one path and have one fall through to the other.

Reply:
I'll be sure to address that in the PR. There's a lot of decisions to be made in #883 

For now, I'd like to keep existing behavior where it was overwritten.

Reply:
And if you still do want to unmount, I think you would then need to factor in how it affects the `after` index if it's a non-NULL, non-zero value.

Reply:
Hmmmm. Let's keep this in draft to see if we're even going to merge #883